rq/worker.py

```diff
@@ -160,11 +160,22 @@ def __init__(self, queues, name=None,
 
     def validate_queues(self):
         """Sanity check for the given queues."""
-        if not iterable(self.queues):
-            raise ValueError('Argument queues not iterable.')
         for queue in self.queues:
             if not isinstance(queue, self.queue_class):
-                raise NoQueueError('Give each worker at least one Queue.')
+                raise NoQueueError('Give each worker at least one Queue.\
```

Comment:
I know this error message has been here from the very start. Doesn't `Give each worker at least one Queue` sound a bit strange? I think we should just change the error message to `{} is not a Queue`.